packages/suite/src/views/wallet/send/Options/BitcoinOptions/Locktime/Locktime.tsx

```diff
@@ -77,32 +109,126 @@ export const Locktime = ({ close }: LocktimeProps) => {
             if (!isInteger(value)) {
                 return translationString('LOCKTIME_IS_NOT_INTEGER');
             }
-            // max unix timestamp * 2 (2147483647 * 2)
-            if (amountBig.gt(4294967294)) {
+            // Bigger numbers denote time, not number of blocks
+            if (amountBig.gte(BTC_LOCKTIME_VALUE)) {
                 return translationString('LOCKTIME_IS_TOO_BIG');
             }
         },
     };
 
-    return (
-        <Card>
+    const { ref: inputRef, ...inputField } = register(datetimeInputName, {
+        onChange: () => {
+            composeTransaction(datetimeInputName);
+        },
+        required: translationString('LOCKTIME_IS_NOT_SET'),
+        validate: {
+            valid: (value: string | undefined) => {
+                if (parseUTCdatetime(value) === undefined) {
+                    return translationString('LOCKTIME_INVALID_DATETIME');
+                }
+            },
+            too_low: (value: string | undefined) => {
+                if (getUnixTime(parseUTCdatetime(value)!)! < BTC_LOCKTIME_VALUE) {
+                    return translationString('LOCKTIME_IS_TOO_LOW');
+                }
```

Comment:
Safer than asserting with `!`:

```suggestion
                const parsedUTCdatetime = value === undefined ? undefined : parseUTCdatetime(value);
                if (parsedUTCdatetime && getUnixTime(parsedUTCdatetime) < BTC_LOCKTIME_VALUE) {
                    return translationString('LOCKTIME_IS_TOO_LOW');
                }
```